src/Worker.js

```diff
@@ -1,5 +1,3 @@
-(() => {
```

Comment:
What was the purpose of the IIFE? To prevent `var` from leaking into the global scope?

Reply:
Yeah, I think it's legacy from when we used to run the `Worker.js` bootstrap inline.

But if it's loading as a node module, which it is right now, then `node` will actually do this for us.